backend/catalogo_arqueologico/piezas/management/commands/importInstitutions.py

```diff
@@ -10,10 +10,22 @@
 logger = logging.getLogger(__name__)
```

Comment:
I just wanted to let you know you forgot to put the comment here. Also, this command does not require specifying the column anymore

Reply:
I would also make mention of the artifact creation performed by this command